packages/web/src/components/search/DataSearch.js

```diff
@@ -378,6 +383,12 @@ class DataSearch extends Component {
 		return null;
 	}
 
+	renderCancelIcon = () => {
+		if (this.props.showClear) {
+			return <CancelSvg />;
+		}
```

Comment:
Use a prop `clearIcon` here if user wants to render a custom icon for clearing the field